### PR TITLE
#4464 update slack references to lf ai & data

### LINKED ISSUE #4464: Update references to slack (odpi->lf ai & data)
We have 6 references to slack.odpi.org in our code base odpi/egeria.
We may also have other references in additional repos used by egeria

All need updating to point to the new LF AI & Data Slack team

cc: @mandy-chessell

### PR DESCRIPTION
**Pull request by planetf1**
Signed-off-by: Nigel Jones <nigel.l.jones+git@gmail.com>

Fixes #4464 

Updates references to slack to point to the LF AI & Data slack team

- The primary link used is  http://slack.lfai.foundation - this is because it is the page which makes it easy to sign up and join. There is a login link on this page which can be accessed by existing team members. We didn't have this distinction before as the slack page itself allowed signup. this isn't the case with LF AI & Data. This decision was made as it's primarily to catch new users - existing users should be familar.
 - I've left the repeated text in place rather than point to the slack.md . I did consider removing dups, but some of these files are standard, and simple .mds. users may grep be editing at cli, and it's going to catch more potential users I think this way (so that they can engage/we can help)

### CODE REVIEW

File: Community-Guide.md [hunk 1]
```diff
@@ -35,16 +35,16 @@ There are two regular online meetings using Zoom each week, both on a Tuesday. A
...
 This creates a recorded discussion of design decisions and discussions that complement the project meetings.
 
-Follow the link above and register with the ODPi slack service using your email address.
+Follow the link above and register with the LF AI & Data slack service using your email address.
 Once signed in you can see all of the active slack channels.
-The main slack channel for the Egeria project is called `#egeria-project`:
+The main slack channel for the Egeria project is called `#egeria-discussion`
 
-[https://odpi.slack.com/messages/CAZDMLTFF](https://odpi.slack.com/messages/CAZDMLTFF]).
+Additional channels are added from time to time as new workgroups and discussion topics are established. For egeria these
```
Comment (davidradl) [inline Community-Guide.md:45]: egeria should be Egeria

File: Community-Guide.md [hunk 2]
```diff
@@ -35,16 +35,16 @@ There are two regular online meetings using Zoom each week, both on a Tuesday. A
...
+channel names will begin with '#egeria', you can also view slack channels from other LF AI & Data projects.
```
Comment (davidradl) [inline Community-Guide.md:46]: comma should be full stop

File: Egeria-Operations.md [hunk 3]
```diff
@@ -124,11 +124,15 @@ Or a meeting may be at an organization's offices that are required to maintain a
...
 This creates a recorded discussion of design decisions and discussions that complement the project meetings.
-This is the main slack channel for the Egeria project:
-[https://odpi.slack.com/messages/CAZDMLTFF](https://odpi.slack.com/messages/CAZDMLTFF]).
-Additional channels will be added as new workgroups and discussion topics are established.
+
+Follow the link above and register with the LF AI & Data slack service using your email address.
+Once signed in you can see all of the active slack channels.
+The main slack channel for the Egeria project is called [#egeria-discussion](https://lfaifoundation.slack.com/archives/C01F40J2XA8)
+
+Additional channels are added from time to time as new workgroups and discussion topics are established. For egeria these
```
Comment (davidradl) [inline Egeria-Operations.md:134]: capital E for Egeria

File: developer-resources/tools/Slack.md [hunk 4]
```diff
@@ -3,24 +3,19 @@
...
+Follow the link above and register with the LF AI & Data slack service using your email address.
 Once signed in you can see all of the active slack channels.
-The main slack channel for the Egeria project is called `#egeria-project`:
-
-[https://odpi.slack.com/messages/CAZDMLTFF](https://odpi.slack.com/messages/CAZDMLTFF]).
-
-Additional channels are added from time to time as new workgroups and discussion topics are established.
+The main slack channel for the Egeria project is called [#egeria-discussion](https://lfaifoundation.slack.com/archives/C01F40J2XA8)
 
+Additional channels are added from time to time as new workgroups and discussion topics are established. For egeria these
```
Comment (davidradl) [inline developer-resources/tools/Slack.md:17]: Capital E for Egeria

File: Community-Guide.md (hunk 1 above)
Comment (planetf1) [reply Community-Guide.md]: fixed

File: Community-Guide.md (hunk 2 above)
Comment (planetf1) [reply Community-Guide.md]: fixed

File: developer-resources/tools/Slack.md (hunk 4 above)
Comment (planetf1) [reply developer-resources/tools/Slack.md]: fixed

File: Egeria-Operations.md (hunk 3 above)
Comment (planetf1) [reply Egeria-Operations.md]: fixed

**Comment by planetf1** [comment]
@davidradl thanks for the review. Fixed up.

**Pull request merged**